UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs

```diff
@@ -14,7 +14,7 @@ public class Liver : BodyPartModification
 		/// <summary>
 		///ReagentContainer which the liver uses to hold reagents it will process. Reagents like alcohol will be broken down into their ethanol reagent via ReactionSet
 		/// </summary>
-		private ReagentContainer processingContainer;
+		[SerializeField] private ReagentContainer processingContainer;
```

Comment:
how was this assigned before?

Reply:
```
public override void SetUpSystems()
		{
			base.SetUpSystems();
			processingContainer = GetComponent<ReagentContainer>();
		}
```

like so

Reply:
I don't quite like having to manually assing the component. whats happening here? There are many of the same component in the same gameobject?

Reply:
I don't like it either, but yes. ReagentContainerBody is first along the list and the container intended for processing is below that in the components list. The stomach also has it's stomachContents reagentContainer manually set as well.